src/components/EditGrower.js

```diff
@@ -149,12 +159,9 @@ const EditGrower = (props) => {
             </Grid>
           ))}
 
-          <TextField
-            className={classes.textInput}
-            label="Grower-entered organization"
-            value={getValue('organization')}
-            disabled
-          />
+          <Typography variant="body1" className={classes.typographyWithMargin}>
+            Organization - {getValue('organization')}
```

Comment:
Let's make the meaning of this clear by changing it to:
> Grower-entered organization:

I'd also suggest making the styling distinct from the value – perhaps a lighter shade of grey – and displaying an indicator when there's no value available.